config.json

```diff
@@ -4,7 +4,10 @@
   "repository": "https://github.com/exercism/xkotlin",
   "active": true,
   "deprecated": [
-
+    "accumulate",
```

Comment:
I can port over `list-ops` if you'd prefer not to have a gap between those exercises being deprecated and the replacement being implemented.